gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/DagActionStore.java

```diff
@@ -45,6 +45,16 @@ class DagAction {
     public FlowId getFlowId() {
       return new FlowId().setFlowGroup(this.flowGroup).setFlowName(this.flowName);
     }
+
+    /**
+     *   Replace flow execution id with agreed upon event time to easily track the flow
+     */
+    public static DagActionStore.DagAction updateFlowExecutionId(DagActionStore.DagAction flowAction,
```

Comment:
this would better belong as a method of the `DagAction` class defined on line 39

Reply:
this is included as a method of the class, it's static because it returns a new dagAction obj